config

```diff
@@ -11,3 +11,7 @@ path /static2 file_handler {
 path /status StatusHandler {}
 
 default NotFoundHandler {}
+
+path /proxy ProxyHandler {
+    root www.ucla.edu;
```

Comment:
Not sure if you attended last discussion section, but the TA and the class together concluded that we do not need to configure the port, although the assignment spec says we do.